app/src/js/components/Header/header.js

```diff
@@ -53,10 +56,18 @@ class Header extends React.Component {
     });
   }
 
+  linkTo (path) {
+    if (path[0] === 'Logs') {
+      const kibanaLink = kibanaAllLogsLink(this.props.cumulusInstance);
+      return <a href={kibanaLink} target="_blank">{path[0]}</a>;
+    } else {
+      return <Link to={path[1]}>{path[0]}</Link>;
+    }
+  }
+
   render () {
     const { authenticated } = this.props.api;
-    const activePaths = paths.filter(pathObj => nav.exclude[pathObj[1].replace('/', '')] !== true);
-
+    const activePaths = paths.filter(pathObj => nav.exclude[pathObj[0]] !== true);
```

Comment:
Is it possible to rename pathObj to something else, because those are arrays (yeah, I know they're technically Objects, but still that's misleading).

Also on this line https://github.com/nasa/cumulus-dashboard/pull/630/files#diff-2403c410025721c7ef089f0d869a3ea1R21. Can you change `/logs` to null? or something to show that it's not being used anymore?

Reply:
Changed `pathObj`

So the `/logs` is used to determine a class name in the `className` method. It breaks the page if it's set to null. I changed it to `logs` which still works but maybe indicates it's no longer a link?